docs/apis/calcite.md

```diff
@@ -0,0 +1,235 @@
+Apache Apex, unified stream processing engine enables application developers to process data at very high throughput with low latency. Although the different types of data have different processing needs, the SQL remains a popular and a generic way for processing data. To ensure that existing ETL developers and developers who are well versed with Database applications adopt stream processing application development with ease, integration of SQL with Apex was needed. Being popular Apache project, Apache Calcite was chosen for the integration and to facilitate the use of SQL with Apex.
+
+# Apex-Calcite Integration
+
+Apache Calcite is a highly customizable engine for parsing and planning queries on relational data from various data sources. Apache Calcite provides the functionality of storage independent query optimization and ways to integrate them into other frameworks which would like to take advantage and expose SQL capability to their users. For details about Apache Calcite, please read at [Apache Calcite Website](https://calcite.apache.org/). 
```

Comment:
"sources. Apache Calcite provides the functionality ... and ways" => sources; it provides storage independent optimization of queries and ways
For details about Apache Calcite => For details {omit "about Apache Calcite" since it is repetitive}